tests/unit/session.test.ts

```diff
@@ -49,72 +99,129 @@ describe('Session Service Unit Tests', () => {
     );
   });
 
-  test('should throw ValidationError if issuer is mismatched', async () => {
-    (jwtVerify as jest.Mock).mockResolvedValue({
-      payload: {
-        iss: 'https://invalid-issuer.com',
-        sub: TEST_USER_ID,
-        name: TEST_FULL_NAME,
-      },
+  test('Invalid Issuer 1', async () => {
+    const jwt = await generateJWT('https://pro-1.frontendapi.corbado.io', 600, {
+      sub: TEST_USER_ID,
+      name: TEST_FULL_NAME,
     });
 
-    await expect(sessionService.validateToken(SHORT_SESSION)).rejects.toThrow(ValidationError);
-    await expect(sessionService.validateToken(SHORT_SESSION)).rejects.toThrow(
-      httpStatusCodes[ValidationErrorNames.InvalidIssuer].description,
+    await expect(sessionService.validateToken(jwt)).rejects.toThrow(ValidationError);
+    await expect(sessionService.validateToken(jwt)).rejects.toThrow(
+      "JWT issuer mismatch (configured trough FrontendAPI: 'https://pro-2.frontendapi.corbado.io', JWT issuer: 'https://pro-1.frontendapi.corbado.io')"
     );
   });
 
-  test('should throw ValidationError if issuer is undefined', async () => {
-    (jwtVerify as jest.Mock).mockResolvedValue({
-      payload: {
-        sub: TEST_USER_ID,
-        name: TEST_FULL_NAME,
-      },
+  test('Invalid Issuer 2', async () => {
```

Comment:
did you check if we have exactly the same number and same "logic implementation" of unit tests compared to the Go SDK? We should have exactly the same.